Automattic/jetpack
# Lazy Images: Start linting lazy-images.js

Fixes #9232

#### Changes proposed in this Pull Request:

* Split the IntersectionObserver polyfill to a separate file, so we can start linting the code we write.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* No behavior should change, just verify things still work. Particularly on whichever browsers we support that need the polyfill (see https://caniuse.com/intersectionobserver?)

#### Proposed changelog entry for your changes:

* None should be necessary.
